### ISSUE TITLE
Add load script for Windows

### ISSUE THREAD

**Issue by monicasarbu** [elastic/packetbeat-dashboards#64]
Merge to the 1.1 branch the PR: https://github.com/elastic/beats-dashboards/pull/57

**Comment by andrewkroh** [comment]
I found an issue with the script #66 so we may want to hold off on merging to 1.1 until it gets fixed.

**Comment by andrewkroh** [comment]
This will need documented in the libbeat Getting Started guide where it shows how to load the dashboards on unix.

**Comment by andrewkroh** [comment]
Also the projects README should be updated to show instructions for Windows.

**Comment by dedemorton** [comment]
@andrewkroh I'm updating the doc to tell users how to download and run the script. Will there be a downloadable zip file somewhere equivalent to what we provide here for other platforms: http://download.elastic.co/beats/dashboards/beats-dashboards-1.1.0.tar.gz

**Comment by andrewkroh** [comment]
A `tar.gz` isn't very Windows friendly so maybe we should only package the dashboards as a zip file. 

@monicasarbu @tsg Any issues with me changing the release script to build a zip instead of tar.gz?

**Comment by monicasarbu** [comment]
@andrewkroh Good point, Andrew. Indeed it is strange to offer tar.gz for Windows. I totally agree that we should provide a zip archive.

**Comment by damm** [comment]
Windows can handle .tar.gz by default now no? I had no problem with Windows 8 in a VM with them

**Comment by dedemorton** [comment]
@monicasarbu @andrewkroh Is there a final resolution on this? If we change the script to build a zip, do we plan to do it before 1.1 ships?

**Comment by dedemorton** [comment]
Just noting that the script has been updated to be a zip through this PR: https://github.com/elastic/beats-dashboards/pull/70
